tick/stateful/eval_function_node.go

```diff
@@ -63,8 +84,8 @@ func (n *EvalFunctionNode) callFunction(scope *Scope, executionState ExecutionSt
 
 	// Look for function on scope
 	if f == nil {
-		if dm := scope.DynamicMethod(n.funcName); dm != nil {
-			f = dynamicMethodFunc{dm: dm}
+		if df := scope.DynamicFunc(n.funcName); df != nil {
+			f = df
 		}
 	}
```

Comment:
Since this logic is repeated above should we make a small helper method for it? Something like `lookupFunction(state, scope)`?

Not super important, just thought of it while looking through.